Editor/Components/GuidedDataCollectorEditor.cs

```diff
@@ -91,10 +155,20 @@ public override void OnInspectorGUI()
             }
 
             serializedObject.ApplyModifiedProperties();
+        }
 
+        private void StepThroughCustomPhalanges(int amt = 1)
```

Comment:
why is this editor logic and not in the monobahaviour?

Reply:
moved over